scripts/tools/GlobalTimestamps.yml

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Tool Experimental Extension for Global Metrics Timestamps"
+version: "1.5"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: "Global Metrics Timestamps Experimental Extension Name"
+version: "1.5"
+name: $T_GLOBAL_METRICS_TIMESTAMPS_EXP_NAME
+value: '"$XT_experimental_global_metrics_timestamps"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Global Metrics Timestamps Experimental Extension Version(s)"
+version: "1.5"
+name: $x_global_metrics_timestamps_exp_version_t
+etors:
+    - name: "1_0"
+      value: "$X_MAKE_VERSION( 1, 0 )"
+      desc: "version 1.0"
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Returns global device timestamps synchronized with metrics timesstamps, optionally mapped to host."
+class: $tMetricGroup
+name: GetTimestampsExp
+decl: static
+details:
+    - "The application may call this function from simultaneous threads."
+params:
+    - type: "$t_metric_group_handle_t"
+      name: hMetricGroup
+      desc: "[in] handle of the metric group"
```

Comment:
this is an array of metric group hangles

Reply:
@matcabral isn't the idea to query for available metric groups with zetMetricGroupGet, where we are already returning an array, and then operate on which metric group individually?

Reply:
correct for metric groups handles collection with zetMetricGroupGet(). However, the idea to support multiple metric group handles and timestamps in the synch API is that different metric groups could have different clock sources. Therefore, user may whant to confirm they are all in synch